code-challenges.test.js

```diff
@@ -48,5 +118,29 @@ const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
+it("takes in an array of 5 numbers and determines whether or not the array is a full house", () => {
+    expect(isHouseFull(hand1)).toEqual(true)
+    expect(isHouseFull(hand2)).toEqual(false)
+    expect(isHouseFull(hand3)).toEqual(false)
+    expect(isHouseFull(hand4)).toEqual(true)
+    })
+})
+
+// output: ReferenceError: isHouseFull is not defined
 
 // b) Create the function that makes the test pass.
+    // pseudo code: 
+        // iterate with for loop
+        // use conditional statement to check if array contains (1,1,1) (2,2,2) etc and (1,1) (2,2) etc. 
+
+        const isHouseFull = (array) => {
+            const count = {}
+            for (const num of array) {
+            count[num] = (count[num] || 0) + 1
+            }
+            return Object.values(count).includes(2) && Object.values(count).includes(3)
+        }
+
+            // output:  PASS  ./code-challenges.test.js
+
+            // notes: relied heavily on online resources for this one. My pseudo code doesn't match it in the slightest. 
```

Comment:
Whenever you find solutions online, make sure you understand exactly what is going on, and then rewrite your pseudo code. If you are not breaking the concepts here down and understanding them, you are only doing yourself a disservice!